dts/arm/st/h7/stm32h7a3.dtsi

```diff
@@ -47,6 +47,11 @@
 			label = "LTDC";
 			status = "disabled";
 		};
+
+		rng: rng@48021800 {
+			nist-config = <0xf00d00>;
+			health-test-magic = <0x72ac>;
```

Comment:
The value of `health-test-magic` should be `0x17590abc`, `0x72ac` is the value of `health-test-config`, which seem to be missing